test/new-e2e/utils/e2e/e2e.go

```diff
@@ -55,6 +265,21 @@ type suiteConstraint[Env any] interface {
 	initSuite(stackName string, stackDef *StackDefinition[Env], options ...func(*Suite[Env]))
 }
 
+// Run run the tests defined in e2eSuite
+//
+// t is an instance of type [*testing.T].
+//
+// e2eSuite is a pointer to a structure with a [e2e.Suite] embbeded struct.
+//
+// stackDef defines the stack definition.
+//
+// options is an optional list of options like [DevMode], [SkipDeleteOnFailure] or [WithStackName].
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Not related to this PR: can we have a type for this options and have the full list ?

Reply:
I don't think so.

Reply:
As there are functions I don't think so.